components/dash-core-components/src/components/Input.react.js

```diff
@@ -66,13 +66,15 @@ export default class Input extends PureComponent {
                 data-dash-is-loading={
                     (loading_state && loading_state.is_loading) || undefined
                 }
+                className={`dash-input ${this.props.className || ''}`}
```

Comment:
This leave an extra space if there is no className.

Reply:
Addressed in 745f0f3